src/vue/forms/VerificationCorporateForm.vue

```diff
@@ -0,0 +1,259 @@
+import FormConfirmation from '@/vue/common/FormConfirmation'
+
+import VerificationFormMixin from '@/vue/mixins/verification-form.mixin'
+
+import { Sdk } from '@/sdk'
+import { ACCOUNT_TYPES } from '@tokend/js-sdk'
+
+import { REQUEST_STATES_STR } from '@/js/const/request-states.const'
+import { BLOB_TYPES } from '@/js/const/blob-types.const'
+
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+import { required, url, integer, minValue } from '@validators'
+
+const MIN_TEAM_SIZE = 1
+
+export default {
+  name: 'verification-corporate-form',
+  components: {
+    FormConfirmation,
+  },
+  mixins: [VerificationFormMixin],
+  data: _ => ({
+    form: {
+      name: '',
+      company: '',
+      headquarters: '',
+      industry: '',
+      foundDate: '',
+      teamSize: '0',
+      website: '',
+    },
+    accountType: ACCOUNT_TYPES.syndicate,
+    MIN_TEAM_SIZE,
+    isConfirming: false,
+  }),
+  validations: {
+    form: {
+      name: { required },
+      company: { required },
+      headquarters: { required },
+      industry: { required },
+      foundDate: { required },
+      teamSize: {
+        required,
+        integer,
+        minValue: minValue(MIN_TEAM_SIZE),
+      },
+      website: { required, url },
+    },
+  },
+  async created () {
+    try {
+      await this.loadKyc()
+    } catch (e) {
+      console.error(e)
+      ErrorHandler.process(e)
+    }
+    if (this.kycState) {
+      this.form = this.convertTemplateToForm(this.kycLatestData)
+      if (this.kycState !== REQUEST_STATES_STR.rejected) {
+        this.disableForm()
+      }
+    }
+  },
+  methods: {
+    async submit () {
+      this.isConfirming = false
+      if (!this.isFormValid()) return
+      this.disableForm()
+      try {
+        const kycBlob = await this.createKycBlob(BLOB_TYPES.kycSyndicate)
+        const operation = this.createKycOperation(kycBlob)
+        await Sdk.horizon.transactions.submitOperations(operation)
+        await this.loadKyc()
+      } catch (e) {
+        console.error(e)
+        ErrorHandler.process(e)
+        this.enableForm()
+      }
+    },
+    convertFormToTemplate () {
+      return {
+        name: this.form.name,
+        company: this.form.company,
+        headquarters: this.form.headquarters,
+        industry: this.form.industry,
+        found_date: this.form.foundDate,
+        team_size: this.form.teamSize,
+        homepage: this.form.website,
+      }
+    },
+    convertTemplateToForm (template) {
+      return {
+        name: template.name,
+        company: template.company,
+        headquarters: template.headquarters,
+        industry: template.industry,
+        foundDate: template.found_date,
+        teamSize: template.team_size,
+        website: template.homepage,
+      }
+    },
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+@import './app-form';
+
+.verification-corporate-form__submit-btn {
+  @include button-raised();
+
+  margin-right: auto;
+  margin-bottom: 2rem;
```

Comment:
why do we need margin-bottom here? the form shouldn’t know anything about external content

Reply:
Resolved, removed